turbinia/turbiniactl.py

```diff
@@ -272,6 +274,13 @@ def main():
   subparsers.add_parser('server', help='Run Turbinia Server')
 
   args = parser.parse_args()
+  if args.config:
+      try:
+          with open("config/turbinia_config.py", "r") as f:
```

Comment:
Thanks for the PR!

One problem with this method of determining where the config file is is that the config can potentially live in many different places, including in a directory that an environment variable points at.  See here for the logic for finding/loading the config file: https://github.com/google/turbinia/blob/eb0a8b09a2e0943eb21b8cacf725d806b4d10f6b/turbinia/config/__init__.py#L28

We could probably refactor `LoadConfig()` in `turbinia/config/__init__.py` out into a `FindConfig()` and a `LoadConfig()` to make this happen.

Reply:
Do we need to even load it a second time? What about just calling `config.LoadConfig()` then just printing out all of the keys in `config`? This wouldn't print comments however.

Alternatively `config.configSource` should point to the path of the config file, so you could just read and print that out.

Reply:
@ericzinnikas Yeah, good points.  I don't think we actually need to load it again then, and I think printing the contents is probably just fine (we might want to filter to only print what keys are in `REQUIRED_VARS` and `OPTIONAL_VARS` though).  It will lose some of the ordering, but I think that's ok.